core/server/api/canary/members.js

```diff
@@ -348,6 +348,52 @@ module.exports = {
         }
     },
 
+    updateSubscription: {
```

Comment:
I think better name to follow existing conventions would be `editSubscription` because we usually use add/edit/destroy names for resource modification ;)

Reply:
Yeah good point, I'll update 😄

Reply:
Done, thanks @naz 🙌